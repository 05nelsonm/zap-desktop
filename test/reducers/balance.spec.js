import balanceReducer, {
  GET_BALANCE,
  RECEIVE_BALANCE
} from '../../app/reducers/balance'

describe('reducers', () => {
  describe('balanceReducer', () => {
    it('should handle initial state', () => {
      expect(balanceReducer(undefined, {})).toMatchSnapshot()
    })
  })
})