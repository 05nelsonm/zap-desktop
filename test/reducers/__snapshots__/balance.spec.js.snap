// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`reducers balanceReducer should handle initial state 1`] = `
Object {
  "balanceLoading": false,
  "channelBalance": null,
  "walletBalance": null,
}
`;
